library/retrieve_conjur_variable.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/python
 
 import os.path
+import ssl
 from ansible.module_utils.basic import *
-from httplib import HTTPConnection
```

Comment:
I believe we should support both HTTP & HTTPS connections.  As a developer working locally, running HTTPS can be very challenging to setup. We can use the Role to configure HTTPS connections with the `conjur_url`, `ssl_certificate`, and `validate_certs` arguments.  If `conjur_url` starts with `https`, we'll need to use `HTTPSConnection`.  If `validate_certs` if false, we can use insecure ssl connections.  The `ssl_certificate` attribute takes an SSL cert to use for HTTPS connections.

What do you think about expanding the `conjur.conf.j2 template to include these additional connection configuration scenarios?  It would allow the plugin and module to make use of the connection setup laid down by the role.

Reply:
Yeah, I already implemented the same logic you wrote :) 
 I am waiting for the previous PRs to be merge to avoid conflicts, when it we be bone I will create the PR with HTTPS & HTTP support 

For meanwhile you can take a look on the branch : Fix_ssl_issue_32